tests/lib/test-utils.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
     if (global.location) {
       return urlUtils.getOrigin(global.location.href);
     }
-    return 'http://localhost:8081';
+    return /^v0\.(?:8|10)/.test(process.version)
```

Comment:
Would it be worth being a little more strict about the version check? What if they used `0.9` or `0.6`?